src/libraries/System.Net.NetworkInformation/ref/System.Net.NetworkInformation.cs

```diff
@@ -123,9 +123,11 @@ public partial class IPAddressInformationCollection : System.Collections.Generic
     public abstract partial class IPGlobalProperties
     {
         protected IPGlobalProperties() { }
+        [System.Runtime.Versioning.UnsupportedOSPlatform("android")]
```

Comment:
I don't see any actual surface area changes here (or other /ref files), but there are `UnsupportedOSPlatform` attribute additions.  These will be perceived as a breaking change to anyone who might have been using these API on these platforms.  This warrants a breaking change notice since we don't typically ship this sort of thing in an in-place servicing update.

Reply:
@ericstj these platforms are new in .net 6 so as far as I know there should be no breaking changes here.

Reply:
Can folks be building against the platforms using 6.0 GA?

Reply:
no, 6.0 GA is not supported for ios/android/etc.

Reply:
I see.  It's up to you if you think customers will hit it or not.  We sometimes document preview->preview breaking changes if we think many folks will hit it.